src/redux/reducer.js

```diff
@@ -0,0 +1,15 @@
+import { XACTIN_LIST_FETCHED } from './actions.js';
+
+const initialState = {};
+
+export const xactionsReducer = (state = initialState, action) => {
```

Comment:
why `xactions` with x at the beginning?)

Reply:
X is commonly used to refer to "cross" because of it's shape of two lines crossing.
Trans - means cross/across. Trans and cross are interchangeable in many cases.